# Type
## Common fields
_Fields common to all schemas in this document_

Property name | Description | Type | Required? | Example 
--- | --- | --- | --- | ---
 describedBy | The URL reference to the schema. | string | no |  |  |  | 
schema_version | The version number of the schema in major.minor.patch format. | string | no | 4.6.1

## Sequence file
_A file of read sequences generated by a sequencing experiment._

Location: type/file/sequence_file.json

Property name | Description | Type | Required? | Object reference? | User friendly name | Allowed values | Example 
--- | --- | --- | --- | --- | --- | --- | --- 
schema_type | The type of the metadata schema entity. | string | yes |  |  | file | 
provenance | Provenance information provided by the system. | object | no | [See   provenance](.md#provenance) |  |  | 
file_core | Core file-level information. | object | yes | [See core  file_core](core.md#file-core) | File core |  | 
read_index | The sequencing read this file represents. | string | yes |  | Read index | read1, read2, read3, read4, index1, index2, single-end, non-indexed | Should be one of: read1, read2, index1, index2
lane_index | The lane that this file was sequenced from. | integer | no |  | Lane index |  | 1
read_length | The length of a sequenced read in this file, in nucleotides. | integer | no |  | Read length |  | 51
insdc_run_accessions | An International Nucleotide Sequence Database Collaboration (INSDC) run accession. | array | no |  | INSDC run accession |  | SRR0000000
sequencing_run_batch | An identifier that indicates which runs were sequenced in the same batch. | string | no |  | Sequencing run batch |  | run1; sequencing_batch_37; NV0087
library_prep_id | A unique ID for the library preparation. | string | no |  | Library preparation ID |  | tech_rep_group_001

## Image file
_Information about an image file generated by an imaging experiment._

Location: type/file/image_file.json

Property name | Description | Type | Required? | Object reference? | User friendly name | Allowed values | Example 
--- | --- | --- | --- | --- | --- | --- | --- 
schema_type | The type of the metadata schema entity. | string | yes |  |  | file | 
provenance | Provenance information provided by the system. | object | no | [See   provenance](.md#provenance) |  |  | 
file_core | Core file-level information. | object | yes | [See core  file_core](core.md#file-core) | File core |  | 

## Supplementary file
_Supplementary files belonging to a project._

Location: type/file/supplementary_file.json

Property name | Description | Type | Required? | Object reference? | User friendly name | Allowed values | Example 
--- | --- | --- | --- | --- | --- | --- | --- 
schema_type | The type of the metadata schema entity. | string | yes |  |  | file | 
provenance | Provenance information provided by the system. | object | no | [See   provenance](.md#provenance) |  |  | 
file_core | Core file-level information. | object | yes | [See core  file_core](core.md#file-core) | File core |  | 
file_description | A short description of the file contents. | string | no |  | File description |  | Protocol for FACS enrichment; Image of spleen sample collection location

## Analysis file
_A file analysis results produced by a secondary analysis pipeline._

Location: type/file/analysis_file.json

Property name | Description | Type | Required? | Object reference? | User friendly name | Allowed values | Example 
--- | --- | --- | --- | --- | --- | --- | --- 
schema_type | The type of the metadata schema entity. | string | yes |  |  | file | 
provenance | Provenance information provided by the system. | object | no | [See   provenance](.md#provenance) |  |  | 
file_core | Core file-level information. | object | yes | [See core  file_core](core.md#file-core) | File core |  | 
matrix_cell_count | Number of cells analyzed in a matrix file. | integer | no |  | Matrix cell count |  | 1; 2100
genome_assembly_version | Name of the genome assembly used to generate this file. | string | yes |  | Genome version | GRCh38, GRCh37, GRCm39, GRCm38, GRCm37, Not Applicable | Should be one of: GRCh38, GRCh37, GRCm39, GRCm38, GRCm37, Not Applicable
genome_patch_version | Patch version of the genome assembly used to generate this file. | string | no |  | Patch version |  | p11; p14

## Reference file
_A reference file used by a secondary reference pipeline._

Location: type/file/reference_file.json

Property name | Description | Type | Required? | Object reference? | User friendly name | Allowed values | Example 
--- | --- | --- | --- | --- | --- | --- | --- 
schema_type | The type of the metadata schema entity. | string | yes |  |  | file | 
provenance | Provenance information provided by the system. | object | no | [See   provenance](.md#provenance) |  |  | 
file_core | Core file-level information. | object | yes | [See core  file_core](core.md#file-core) | File core |  | 
ncbi_taxon_id | A taxonomy ID (taxonID) from NCBI. | integer | yes |  | NCBI taxon ID |  | 9606; 10090
genus_species | The scientific binomial name for the species of this reference. | object | yes | [See module  species_ontology](module.md#species-ontology) | Genus species |  | 
reference_type | The type of the reference file. | string | yes |  | Reference type | genome sequence, transcriptome sequence, annotation reference, transcriptome index, genome sequence index | Should be one of: genome sequence, transcriptome sequence, annotation reference, transcriptome index, or genome sequence index.
assembly_type | The assembly type of the genome reference file. | string | yes |  | Genome assembly type | primary assembly, complete assembly, patch assembly | Should be one of: primary assembly, complete assembly, or patch assembly.
reference_version | The genome version of the reference file. | string | yes |  | Reference version |  | GencodeV27; Ensembl 87

## Protocol
_Information about the protocol._

Location: type/protocol/protocol.json

Property name | Description | Type | Required? | Object reference? | User friendly name | Allowed values | Example 
--- | --- | --- | --- | --- | --- | --- | --- 
schema_type | The type of the metadata schema entity. | string | yes |  |  | protocol | 
provenance | Provenance information provided by the system. | object | no | [See   provenance](.md#provenance) |  |  | 
protocol_core | Core protocol-level information. | object | yes | [See core  protocol_core](core.md#protocol-core) | Protocol core |  | 
type | The type of protocol. | object | no | [See module  process_type_ontology](module.md#process-type-ontology) | Protocol type |  | sample enrichment

## Sequencing protocol
_Information about the sequencing protocol._

Location: type/protocol/sequencing/sequencing_protocol.json

Property name | Description | Type | Required? | Object reference? | User friendly name | Allowed values | Example 
--- | --- | --- | --- | --- | --- | --- | --- 
schema_type | The type of the metadata schema entity. | string | yes |  |  | protocol | 
provenance | Provenance information provided by the system. | object | no | [See   provenance](.md#provenance) |  |  | 
protocol_core | Core protocol-level information. | object | yes | [See core  protocol_core](core.md#protocol-core) | Protocol core |  | 
instrument_manufacturer_model | The manufacturer and model of the sequencer. | object | yes | [See module  instrument_ontology](module.md#instrument-ontology) | Instrument manufacturer and model |  | 
local_machine_name | Local name for the particular machine used for sequencing. | string | no |  | Local machine name |  | Machine1; HSMA-20
paired_end | Whether the sequenced molecule was sequenced from both ends. | boolean | yes |  | Paired end |  | Should be one of: yes, or no.
method | The general method for sequencing. | object | yes | [See module  sequencing_ontology](module.md#sequencing-ontology) | Sequencing method |  | 
10x | Fields specific for 10x experiments. | object | no | [See module  10x](module.md#10x) | 10x-specific |  | 

## Library preparation protocol
_Information about how a sequencing library was prepared._

Location: type/protocol/sequencing/library_preparation_protocol.json

Property name | Description | Type | Required? | Object reference? | User friendly name | Allowed values | Example 
--- | --- | --- | --- | --- | --- | --- | --- 
schema_type | The type of the metadata schema entity. | string | yes |  |  | protocol | 
provenance | Provenance information provided by the system. | object | no | [See   provenance](.md#provenance) |  |  | 
protocol_core | Core protocol-level information. | object | yes | [See core  protocol_core](core.md#protocol-core) | Protocol core |  | 
cell_barcode | Information about cell identifier barcodes. | object | no | [See module  barcode](module.md#barcode) | Cell barcode |  | 
spatial_barcode | Information about spatial barcodes. | object | no | [See module  barcode](module.md#barcode) | Spatial barcode |  | 
input_nucleic_acid_molecule | Starting nucleic acid molecule isolated for sequencing. | object | yes | [See module  biological_macromolecule_ontology](module.md#biological-macromolecule-ontology) | Input nucleic acid molecule |  | 
nucleic_acid_source | Source cells or organelles from which nucleic acid molecules were collected. | string | yes |  | Nucleic acid source | bulk cell, single cell, single nucleus, bulk nuclei, mitochondria | Should be one of: single cell, bulk cell, single nucleus, bulk nuclei, or mitochondria.
library_construction_method | The general method for sequencing library construction. | object | yes | [See module  library_construction_ontology](module.md#library-construction-ontology) | Library construction method |  | 
library_construction_kit | Name of kit used to construct the sequencing library. | object | no | [See module  purchased_reagents](module.md#purchased-reagents) | Library construction kit |  | 
nucleic_acid_conversion_kit | Name of kit used to convert RNA to DNA for sequencing. | object | no | [See module  purchased_reagents](module.md#purchased-reagents) | Nucleic acid conversion kit |  | 
end_bias | The type of tag or end bias the library has. | string | yes |  | End bias | 3 prime tag, 3 prime end bias, 5 prime tag, 5 prime end bias, full length | Should be one of: 3 prime tag, 3 prime end bias, 5 prime tag, 5 prime end bias, or full length.
primer | Primer used for cDNA synthesis from RNA. | string | no |  | Primer | poly-dT, random | Should be one of: poly-dT, or random.
strand | Library strandedness. | string | yes |  | Strand | first, second, unstranded, not provided | Should be one of: first, second, unstranded, or not provided.
spike_in_kit | Information about a spike-in kit. | object | no | [See module  purchased_reagents](module.md#purchased-reagents) | Spike-in kit |  | 
spike_in_dilution | Dilution of spike-in. | integer | no |  | Spike-in dilution |  | 100
umi_barcode | Information about unique molecular identifier (UMI) barcodes. | object | no | [See module  barcode](module.md#barcode) | UMI barcode |  | 
library_preamplification_method | The method used to amplify RNA prior to adaptor ligation. | object | no | [See module  library_amplification_ontology](module.md#library-amplification-ontology) | Library pre-amplification method |  | 
cdna_library_amplification_method | The method used to amplify cDNA library prior to sequencing. | object | no | [See module  library_amplification_ontology](module.md#library-amplification-ontology) | cDNA library amplification method |  | 
nominal_length | Average (insert) size of the fragments being sequenced. | integer | no |  | Nominal length |  | 250
nominal_sdev | Standard deviation of the (insert) size of the fragments being sequenced. | integer | no |  | Nominal standard deviation |  | 30

## Analysis protocol
_Information about the analysis protocol._

Location: type/protocol/analysis/analysis_protocol.json

Property name | Description | Type | Required? | Object reference? | User friendly name | Allowed values | Example 
--- | --- | --- | --- | --- | --- | --- | --- 
schema_type | The type of the metadata schema entity. | string | yes |  |  | protocol | 
provenance | Provenance information provided by the system. | object | no | [See   provenance](.md#provenance) |  |  | 
protocol_core | Core protocol-level information. | object | yes | [See core  protocol_core](core.md#protocol-core) | Protocol core |  | 
type | The type of protocol. | object | yes | [See module  process_type_ontology](module.md#process-type-ontology) | Protocol type |  | 
computational_method | A URI to a versioned workflow and versioned execution environment in a GA4GH-compliant repository. | string | no |  | Computational method |  | SmartSeq2SingleCell; 10x
matrix | Information related to protocols that output a matrix. | object | no | [See module  matrix](module.md#matrix) | Matrix |  | 
alignment_software | Name of alignment software used to map FASTQ files to reference genome. | string | no |  | Alignment software |  | Cellranger; kallisto bustools; GSNAP; STAR
alignment_software_version | Version of alignment software used to map FASTQ files to reference genome. | string | no |  | Alignment software version |  | v2.0.1; 2.4.2a; v0.45.2
gene_annotation_version | The Ensembl release version accession number or NCBI RefSeq assembly version used for gene annotation. | string | no |  | Gene annotation version |  | v110; GCF_000001405.40; GCF_000001635.27
intron_inclusion | Whether introns were included in the alignment process during read counting. | boolean | no |  | Intron inclusion |  | Should be one of: yes, or no.

## Aggregate generation protocol
_Information about how cultured cells are developed into cell aggregates._

Location: type/protocol/biomaterial_collection/aggregate_generation_protocol.json

Property name | Description | Type | Required? | Object reference? | User friendly name | Allowed values | Example 
--- | --- | --- | --- | --- | --- | --- | --- 
schema_type | The type of the metadata schema entity. | string | yes |  |  | protocol | 
provenance | Provenance information provided by the system. | object | no | [See   provenance](.md#provenance) |  |  | 
protocol_core | Core protocol-level information. | object | yes | [See core  protocol_core](core.md#protocol-core) | Protocol core |  | 
formation_method | Method used to form cell aggregates. | string | yes |  | Aggregate formation method |  | rocking; suspension cultures; hanging drops; spinner flasks
cell_uniformity | Description of the cell aggregates uniformity after formation. | string | no |  | Aggregate cell uniformity |  | Mostly homogenous embryoid bodies of variable cell numbers; Low-homogeneity in morphology and size

## Enrichment protocol
_Information about how a biomaterial is enriched for a feature or characteristic of interest._

Location: type/protocol/biomaterial_collection/enrichment_protocol.json

Property name | Description | Type | Required? | Object reference? | User friendly name | Allowed values | Example 
--- | --- | --- | --- | --- | --- | --- | --- 
schema_type | The type of the metadata schema entity. | string | yes |  |  | protocol | 
provenance | Provenance information provided by the system. | object | no | [See   provenance](.md#provenance) |  |  | 
protocol_core | Core protocol-level information. | object | yes | [See core  protocol_core](core.md#protocol-core) | Protocol core |  | 
method | The method by which enrichment was achieved. | object | yes | [See module  process_type_ontology](module.md#process-type-ontology) | Enrichment method |  | 
markers | A list of markers used to enrich for or against certain cells. | string | no |  | Markers |  | CD4+ CD8-; HLA-G+
minimum_size | Minimum cell or organelle size passing selection, in microns. | number | no |  | Minimum size selected |  | 70
maximum_size | Maximum cell or organelle size passing selection, in microns. | number | no |  | Maximum size selected |  | 90

## Dissociation protocol
_Information about the dissociation protocol used to separate individual cells or nuclei._

Location: type/protocol/biomaterial_collection/dissociation_protocol.json

Property name | Description | Type | Required? | Object reference? | User friendly name | Allowed values | Example 
--- | --- | --- | --- | --- | --- | --- | --- 
schema_type | The type of the metadata schema entity. | string | yes |  |  | protocol | 
provenance | Provenance information provided by the system. | object | no | [See   provenance](.md#provenance) |  |  | 
protocol_core | Core protocol-level information. | object | yes | [See core  protocol_core](core.md#protocol-core) | Protocol core |  | 
method | How cells or organelles were dissociated. | object | yes | [See module  process_type_ontology](module.md#process-type-ontology) | Dissociation method |  | 
reagents | A list of purchased reagents used in this protocol. | array | no | [See module  purchased_reagents](module.md#purchased-reagents) | Purchased reagents |  | 
digestion_time | Time of digestion of the specimen. | integer | no |  | Digestion time |  | 2; 5; 10
digestion_time_unit | The unit in which the digestion time is expressed. | object | no | [See module  time_unit_ontology](module.md#time-unit-ontology) | Digestion time unit |  | 
digestion_temperature | Temperature of digestion in Celsius or in general terms (i.e. cold, warm, room temperature). | number or string | no |  | Digestion temperature | frozen, cold, room temperature, warm | 4; 37; cold
digestion_solution | Enzyme(s) or reagent(s) of the solution that was used for the digestion of the specimen. | array | no |  | Digestion solution | Citric-acid based, Detergent-based, Lysis buffer, Accutase, Collagenase, Collagenase A, Collagenase D, Collagenase I, Collagenase II, Collagenase IV, Dispase, Dispase II, DNAse I, Elastase, Hyaluronidase, Liberase, Papain, Protease, TrypLE, Trypsin | 

## iPSC induction protocol
_Information about how a biomaterial is treated to become an induced pluripotent stem cell._

Location: type/protocol/biomaterial_collection/ipsc_induction_protocol.json

Property name | Description | Type | Required? | Object reference? | User friendly name | Allowed values | Example 
--- | --- | --- | --- | --- | --- | --- | --- 
schema_type | The type of the metadata schema entity. | string | yes |  |  | protocol | 
provenance | Provenance information provided by the system. | object | no | [See   provenance](.md#provenance) |  |  | 
protocol_core | Core protocol-level information. | object | yes | [See core  protocol_core](core.md#protocol-core) | Protocol core |  | 
method | Induction method applied to primary cell culture to induce pluripotent stem cell generation. | string | yes |  | Induction method | lentivirus, sendai virus, Gun particle, piggyBac transposon, miRNA viral, adenovirus, cre-loxP, plasmid, retroviral | Should be one of: lentivirus, sendai virus, Gun particle, piggyBac transposon, miRNA viral, adenovirus, cre-loxP, plasmid, or retroviral.
reprogramming_factors | Reprogramming factors added to primary cell culture to induce pluripotency. | string | no |  | Reprogramming factors |  | POU5F1; SOX2; KLF4; c-MYC
ipsc_induction_kit | Kit used to induce pluripotent stem cell generation. | object | no | [See module  purchased_reagents](module.md#purchased-reagents) | Induction kit |  | 
pluripotency_test | Description of how pluripotency was tested in induced pluripotent stem cells. | string | no |  | Pluripotency test |  | Teratoma formation; PluriTest; Rt-PCR
percent_pluripotency | Percent of iPSCs that passed the pluripotency test. | number | no |  | Percent pluripotency |  | 97.2
pluripotency_vector_removed | Whether a viral vector was removed after induction. | string | no |  | Pluripotent vector removed | yes, no, unknown | Should be one of: yes, no, or unknown.
reagents | Additional purchased reagents used in this protocol. | array | no | [See module  purchased_reagents](module.md#purchased-reagents) | Additional reagents |  | 

## Collection protocol
_Information about the biomaterial collection protocol._

Location: type/protocol/biomaterial_collection/collection_protocol.json

Property name | Description | Type | Required? | Object reference? | User friendly name | Allowed values | Example 
--- | --- | --- | --- | --- | --- | --- | --- 
schema_type | The type of the metadata schema entity. | string | yes |  |  | protocol | 
provenance | Provenance information provided by the system. | object | no | [See   provenance](.md#provenance) |  |  | 
protocol_core | Core protocol-level information. | object | yes | [See core  protocol_core](core.md#protocol-core) | Protocol core |  | 
method | Method used to collect the biomaterial. | object | yes | [See module  process_type_ontology](module.md#process-type-ontology) | Collection method |  | 
procedure | Procedure used to collect the specimen from the organism. | string | no |  | Procedure |  | colonoscopy; venipuncture; laparoscopic surgery
reagents | A list of purchased reagents used in this protocol. | array | no | [See module  purchased_reagents](module.md#purchased-reagents) | Protocol reagents |  | 

## Differentiation protocol
_Information about how a cell is differentiated to a desired cell type or organoid._

Location: type/protocol/biomaterial_collection/differentiation_protocol.json

Property name | Description | Type | Required? | Object reference? | User friendly name | Allowed values | Example 
--- | --- | --- | --- | --- | --- | --- | --- 
schema_type | The type of the metadata schema entity. | string | yes |  |  | protocol | 
provenance | Provenance information provided by the system. | object | no | [See   provenance](.md#provenance) |  |  | 
protocol_core | Core protocol-level information. | object | yes | [See core  protocol_core](core.md#protocol-core) | Protocol core |  | 
method | Method applied to cell culture to induce a specific differentiation response. | string | yes |  | Differentiation method |  | Embryoid Body; Monolayer; Inductive Co-Culture
media | Culture media used to induce a specific differentiation response. | string | no |  | Differentiation media |  | RPMI 1640 + B27; Neurobasal Media; StemPro-34 Serum-Free Medium
small_molecules | Small molecules added to stem cell medium to induce a specific differentiation response. | string | no |  | Small molecules |  | Retinoic Acid; CHIR99021 (GSK-inhibitor); Activin A; BMP4
target_cell_yield | Percent of target cells obtained after directed differentiation of origin cell. | number | no |  | Percent target cell yield |  | 95
reagents | A list of purchased reagents used in the differentiation protocol. | array | no | [See module  purchased_reagents](module.md#purchased-reagents) | Differentiation reagents |  | 
target_pathway | Targeted pathway for specific differentiation response. | string | no |  | Target pathway |  | Wnt pathway; Hedgehog signaling pathway
validation_method | Method used to validate origin cell successfully differentiated to target cell. | string | no |  | Differentiation validation method |  | Pancreatic Cell DTZ Detection Assay; qPCR; Flow Cytometry; Immunocytochemistry Staining
validation_result | Result confirming successful differentiation to target cell type. | string | no |  | Validation result |  | CD103 Positive; Nestin Positive; HCN4 Positive; CD11C Negative

## Treatment protocol
_Information about how cells are subjected to a treatment or stimulus, such as an immune activator or a drug._

Location: type/protocol/biomaterial_collection/treatment_protocol.json

Property name | Description | Type | Required? | Object reference? | User friendly name | Allowed values | Example 
--- | --- | --- | --- | --- | --- | --- | --- 
schema_type | The type of the metadata schema entity. | string | yes |  |  | protocol | 
provenance | Provenance information provided by the system. | object | no | [See   provenance](.md#provenance) |  |  | 
protocol_core | Core protocol-level information. | object | yes | [See core  protocol_core](core.md#protocol-core) | Protocol core |  | 
method | Method applied to cell culture to induce a specific treatment response. | array | yes | [See module  treatment_method_ontology](module.md#treatment-method-ontology) | Treatment method |  | 
media | Culture media used to induce a specific treatment response. | string | no |  | Treatment media |  | RPMI 1640; Complete Medium
reagents | A list of purchased reagents used in the treatment protocol. | array | no | [See module  purchased_reagents](module.md#purchased-reagents) | Treatment reagents |  | 
target_pathway | Targeted pathway for specific treatment response. | array | no | [See module  target_pathway_ontology](module.md#target-pathway-ontology) | Target pathway |  | insulin signalling pathway; immune cell activation; chemokine signalling pathway.

## Imaging preparation protocol
_Information about the preparation protocol of the imaged specimen used in an imaging experiment._

Location: type/protocol/imaging/imaging_preparation_protocol.json

Property name | Description | Type | Required? | Object reference? | User friendly name | Allowed values | Example 
--- | --- | --- | --- | --- | --- | --- | --- 
schema_type | The type of the metadata schema entity. | string | yes |  |  | protocol | 
provenance | Provenance information provided by the system. | object | no | [See   provenance](.md#provenance) |  |  | 
protocol_core | Core protocol-level information. | object | yes | [See core  protocol_core](core.md#protocol-core) | Protocol core |  | 
fresh_slicing_method | The method by which fresh tissue was sliced. | string | no |  | Fresh slicing method |  | vibrotome
imaged_slice_thickness | Thickness of the imaged slice in micrometres. | number | no |  | Imaged slice thickness |  | 10
final_slicing_method | The method by which the final slice was obtained. | string | no |  | Final slicing method |  | cryosectioning
post_resection_interval | Length of time between surgical resection and fresh slicing of tissue. | number | no |  | Time between resection and fresh slicing |  | 5
post_resection_interval_unit | The unit of time in which the post resection interval is expressed. | object | no | [See module  time_unit_ontology](module.md#time-unit-ontology) | Post resection interval time unit |  | day
pre_final_slice_preservation_method | Tissue preservation method used prior to final slicing. | object | no | [See module  preservation_storage](module.md#preservation-storage) | Pre-final slice preservation method |  | freezing
post_final_slicing_interval | Length of time between secondary slicing and hybridization. | number | no |  | Post final slicing interval |  | 7
post_final_slicing_interval_unit | The unit of time in which the post final slicing interval is expressed. | object | no | [See module  time_unit_ontology](module.md#time-unit-ontology) | Post final slicing interval time unit |  | day
fiducial_marker | Fiducial markers for the alignment of images taken across multiple rounds of imaging. | string | no |  | Fiducial marker |  | beads
expansion_factor | Factor by which the imaged tissue was expanded in one dimension. | number | no |  | Expansion factor |  | 3
permeabilisation_time | The permeabilisation time in time units that the tissue was exposed to. | number | no |  | Permeabilisation time |  | 12
permeabilisation_time_unit | The unit in which permeabilisation time is expressed. | object | no | [See module  time_unit_ontology](module.md#time-unit-ontology) | Permeabilisation time unit |  | 

## Imaging Protocol
_Information about the imaging protocol._

Location: type/protocol/imaging/imaging_protocol.json

Property name | Description | Type | Required? | Object reference? | User friendly name | Allowed values | Example 
--- | --- | --- | --- | --- | --- | --- | --- 
schema_type | The type of the metadata schema entity. | string | yes |  |  | protocol | 
provenance | Provenance information provided by the system. | object | no | [See   provenance](.md#provenance) |  |  | 
protocol_core | Core protocol-level information. | object | yes | [See core  protocol_core](core.md#protocol-core) | Protocol core |  | 
microscope_setup_description | Description of the microscope setup. | string | no |  | Microscope setup description |  | Motorized stage (SCAN IM 112 x 74, Marzhauser), sCMOS camera (Zyla 4.2; Andor), 60x, Plan-Apo, 1.3 NA, silicone oil objective (UPLSAPO60XS2; Olympus).
microscopy_technique | The type of microscopy. | object | yes | [See module  microscopy_ontology](module.md#microscopy-ontology) | Microscopy technique |  | 
magnification | Magnification of the objective used for imaging. | string | yes |  | Magnification |  | 60x; 100x
numerical_aperture | Numerical aperture of the objective. | number | no |  | Numerical aperture |  | 1.3; 0.75
immersion_medium_type | Immersion medium used for imaging. | string | no |  | Immersion medium |  | oil; air
immersion_medium_refractive_index | Refractive index of the immersion medium used for imaging. | number | no |  | Refractive index of the immersion medium |  | 1.5; 1
pixel_size | Pixel size in nanometers. | number | no |  | Pixel size |  | 109
number_of_tiles | Number of XY tiles in the experiment. | integer | no |  | Number of tiles |  | 2000
tile_size_y | Y size of the tile in micrometers. | number | no |  | Tile size Y |  | 100
tile_size_x | X size of the tile in micrometers. | number | no |  | Tile size X |  | 100
z_stack_step_size | Z-stack step size in nanometers. | number | no |  | Z stack step size |  | 200
number_of_z_steps | Number of steps in a Z stack. | integer | no |  | Number of Z steps |  | 40
overlapping_tiles | Whether tiles were collected with overlap. | string | no |  | Overlapping tiles | yes, no | Should be one of: yes, or no.
channel | Information about each channel used in the imaging protocol. | array | no | [See module  channel](module.md#channel) | Channel |  | 
probe | Information about each probe in the imaging experiment. | array | no | [See module  probe](module.md#probe) | Probe |  | 

## Project
_A project entity contains information about the overall project._

Location: type/project/project.json

Property name | Description | Type | Required? | Object reference? | User friendly name | Allowed values | Example 
--- | --- | --- | --- | --- | --- | --- | --- 
schema_type | The type of the metadata schema entity. | string | yes |  |  | project | 
provenance | Provenance information provided by the system. | object | no | [See   provenance](.md#provenance) |  |  | 
project_core | Core project-level information. | object | yes | [See core  project_core](core.md#project-core) | Project core |  | 
contributors | People contributing to any aspect of the project. | array | no | [See module  contact](module.md#contact) | Contributors |  | 
supplementary_links | External link(s) pointing to code, supplementary data files, or analysis files associated with the project which will not be uploaded. | array | no |  | Supplementary link(s) |  | https://github.com/czbiohub/tabula-muris; http://celltag.org/
publications | Publications resulting from this project. | array | no | [See module  publication](module.md#publication) | Publications |  | 
hca_bionetworks | HCA Bionetworks and Atlases the project is associated with | array | no | [See module  hca_bionetwork](module.md#hca-bionetwork) | HCA Bionetwork(s) |  | 
insdc_project_accessions | An International Nucleotide Sequence Database Collaboration (INSDC) project accession. | array | no |  | INSDC project accession |  | SRP000000
ega_accessions | A list of accessions referring to EGA (European Genome-Phenome Archive) datasets or studies. | array | no |  | EGA Study/Dataset Accession(s) |  | EGAS00000000001; EGAD00000000002
dbgap_accessions | A list of database of Genotypes and Phenotypes (dbGaP) study accessions. | array | no |  | dbGaP Study Accession(s) |  | phs001997.v1.p1; phs001836
geo_series_accessions | A Gene Expression Omnibus (GEO) series accession. | array | no |  | GEO series accession |  | GSE00000
array_express_accessions | An ArrayExpress accession. | array | no |  | ArrayExpress accession |  | E-AAAA-00
insdc_study_accessions | An International Nucleotide Sequence Database Collaboration (INSDC) study accession. | array | no |  | INSDC study accession |  | PRJNA000000
biostudies_accessions | A BioStudies study accession. | array | no |  | BioStudies accession |  | S-EXMP1; S-HCAS33
funders | Funding source(s) supporting the project. | array | yes | [See module  funder](module.md#funder) | Funding source(s) |  | 
estimated_cell_count | An estimated number of cells in this project | integer | no |  | Estimated cell count |  | 10000; 2100000
data_use_restriction | Data use restrictions that apply to the project. | string | yes |  | Data use restriction | NRES, GRU, GRU-NCU | GRU
duos_id | A DUOS dataset id. | string | no |  | DUOS ID |  | DUOS-000108; DUOS-000114

