# Type
## sequence_file
_A file of read sequences generated by a sequencing experiment._

Location: type/file/sequence_file.json

Property name | Description | Type | Required? | User friendly name | Example 
--- | --- | --- | --- | --- | --- 
$schema | The URL reference to the schema. | string | yes |  | 
schema_version | The version number of the schema in major.minor.patch format. | string | no |  | 4.6.1
schema_type | The type of the metadata schema entity. | string | yes |  | 
file_core | Core file-level information. | string | yes |  | 
read_index | Whether the read file contains the read1, read2, index1, or index2 part of the sequencing read. If read file represents a single-end, non-indexed library, indicate that here. | string | yes | Read index | read1
lane_index | The index of the lane that this file was sequenced from. | integer | no | Lane index | 1

## protocol
_Information about the protocol_

Location: type/protocol/protocol.json

Property name | Description | Type | Required? | User friendly name | Example 
--- | --- | --- | --- | --- | --- 
$schema | The URL reference to the schema. | string | yes |  | 
schema_version | The version number of the schema in major.minor.patch format. | string | no |  | 4.6.1
schema_type | The type of the metadata schema entity. | string | yes |  | 
protocol_core | Core protocol-level information. |  | yes |  | 
protocol_type | The type of protocol. Must be a term from the Experimental Factor Ontology. |  | no | Protocol type | 

## sequencing_protocol
_Information about the sequencing protocol_

Location: type/protocol/sequencing/sequencing_protocol.json

Property name | Description | Type | Required? | User friendly name | Example 
--- | --- | --- | --- | --- | --- 
$schema | The URL reference to the schema. | string | yes |  | 
schema_version | The version number of the schema in major.minor.patch format. | string | no |  | 4.6.1
schema_type | The type of the metadata schema entity. | string | yes |  | 
protocol_core | Core protocol-level information. |  | yes |  | 
protocol_type | The type of protocol. Must be a term from the Experimental Factor Ontology. |  | no | Protocol type | 

## analysis_protocol
_Information about the analysis protocol_

Location: type/protocol/analysis/analysis_protocol.json

Property name | Description | Type | Required? | User friendly name | Example 
--- | --- | --- | --- | --- | --- 
$schema | The URL reference to the schema. | string | yes |  | 
schema_version | The version number of the schema in major.minor.patch format. | string | no |  | 4.6.1
schema_type | The type of the metadata schema entity. | string | yes |  | 
protocol_core | Core protocol-level information. |  | yes |  | 
protocol_type | The type of protocol. Must be a term from the Experimental Factor Ontology. |  | no | Protocol type | 

## imaging_protocol
_Information about the imaging protocol_

Location: type/protocol/imaging/imaging_protocol.json

Property name | Description | Type | Required? | User friendly name | Example 
--- | --- | --- | --- | --- | --- 
$schema | The URL reference to the schema. | string | yes |  | 
schema_version | The version number of the schema in major.minor.patch format. | string | no |  | 4.6.1
schema_type | The type of the metadata schema entity. | string | yes |  | 
protocol_core | Core protocol-level information. |  | yes |  | 
protocol_type | The type of protocol. Must be a term from the Experimental Factor Ontology. |  | no | Protocol type | 

## biomaterial_protocol
_Information about the biomaterial collection protocol_

Location: type/protocol/biomaterial/biomaterial_collection_protocol.json

Property name | Description | Type | Required? | User friendly name | Example 
--- | --- | --- | --- | --- | --- 
$schema | The URL reference to the schema. | string | yes |  | 
schema_version | The version number of the schema in major.minor.patch format. | string | no |  | 4.6.1
schema_type | The type of the metadata schema entity. | string | yes |  | 
protocol_core | Core protocol-level information. |  | yes |  | 
protocol_type | The type of protocol. Must be a term from the Experimental Factor Ontology. |  | no | Protocol type | 

## project
_A project contains information about the overall project._

Location: type/project/project.json

Property name | Description | Type | Required? | User friendly name | Example 
--- | --- | --- | --- | --- | --- 
$schema | The URL reference to the schema. | string | yes |  | 
schema_version | The version number of the schema in major.minor.patch format. | string | no |  | 4.6.1
schema_type | The type of the metadata schema entity. | string | yes |  | 
project_core | Core project-level information. |  | no |  | 
contributors | List of people contributing to any aspect of the project. | array | no |  | List of authors for a publication resulting from this project.
supplementary_files | Project-level supplementary files. | array | no | Supplementary files | Experimental design documents, electronic notebook files, manuscripts in preparation, etc.
publications | A list of publications resulting from this project. | array | no |  | 
insdc_project | An INSDC (International Nucleotide Sequence Database Collaboration) project accession. Can be from the DDBJ, EMBL-EBI, or NCBI. Accession must start with DRP, ERP, or SRP. | string | no | INSDC project accession | 
geo_series | An NCBI GEO series accession. | string | no | GEO series accession | 
array_express_investigation | An EBI ArrayExpress investigation accession. | string | no | ArrayExpress accession | 
insdc_study | An INSDC (International Nucleotide Sequence Database Collaboration) study accession. Can be from the DDBJ, EMBL-EBI, or NCBI. Accession must start with PRJE, PRJN, or PRJD. | string | no | INSDC study accession | 
related_projects | A list of other projects that may be logically grouped with this one. | array | no | Related projects | 

## specimen_from_organism
_Information about the specimen that was extracted from the organism._

Location: type/biomaterial/specimen_from_organism.json

Property name | Description | Type | Required? | User friendly name | Example 
--- | --- | --- | --- | --- | --- 
$schema | The URL reference to the schema. | string | yes |  | 
schema_version | The version number of the schema in major.minor.patch format. | string | no |  | 4.6.1
schema_type | The type of the metadata schema entity. | string | yes |  | 
biomaterial_core | Core biomaterial-level information. |  | yes |  | 
genus_species | The scientific binomial name for the species of the biomaterial. | array | no | Genus species | Homo sapiens
organ | The organ that the biomaterial came from. Blood and connective tissue are considered organs. |  | yes | Organ | Blood
organ_part | A term for a specific part of the organ that the biomaterial came from. |  | yes | Organ part | Umbilical cord blood
state_of_specimen | State of the specimen at the time of collection/removal. |  | no |  | 
preservation_storage | Information relating to how a biomaterial was preserved and/or stored over a period of time. |  | no |  | 
collection_time | When the biomaterial was collected, in date-time format, yyyy-mm-ddThh:mm:ssZ. | string | no | Time of collection | 

## organism
_Information about the organism from which a specimen was collected._

Location: type/biomaterial/organism.json

Property name | Description | Type | Required? | User friendly name | Example 
--- | --- | --- | --- | --- | --- 
$schema | The URL reference to the schema. | string | yes |  | 
schema_version | The version number of the schema in major.minor.patch format. | string | no |  | 4.6.1
schema_type | The type of the metadata schema entity. | string | yes |  | 
biomaterial_core | Core biomaterial-level information. |  | yes |  | 
human_specific | Fields specific to human (homo sapiens) organisms. |  | no |  | 
mus_musculus_specific | Fields specific to mouse (mus musculus) organisms. |  | no |  | 
death | Information about conditions of death of an organism. |  | no |  | 
medical_history | Information about the medical history of an organism. |  | no |  | 
genus_species | The scientific binomial name for the species of the biomaterial. | array | no | Genus species | Homo sapiens
organism_age | Age in age_units. Measured since birth. | number | no | Organism age | 
organism_age_unit | The unit in which age is expressed. Must be one of hour, day, week, month, or year. |  | no | Organism age unit | 
development_stage | A classification of the developmental stage of the organism. |  | no | Development stage | 
disease | Short description of disease status of the organism. | array | no | Disease | 
familial_relationship | Information about other organisms related to this organism. | array | no |  | 
gestational_age | Gestational age in gestational_age_units. Measured since fertilization. | number | no | Gestational age | 
gestational_age_unit | The unit in which gestational age is expressed. Must be one of hour, day, week, month, or year. |  | no | Gestational age unit | 
height | Height of organism in height units. | number | no | Height | 
height_unit | The unit in which height is expressed. Must be one of centimeters, meters, or inches. |  | no | Height unit | 
is_living | Yes if organism is living at time of biomaterial collection. No otherwise. | boolean | yes | Is living? | 
biological_sex | The biological sex of the organism. Should be one of male, female, mixed, or unknown. | string | yes | Biological sex | 
weight | Weight of organism in kilograms. | number | no | Weight | 
weight_unit | The unit in which weight is expressed. Must be one of gram or kilogram. |  | no | Weight unit | 

## cell_suspension
_Information about the cell suspension derived from the collected or cultured specimen_

Location: type/biomaterial/cell_suspension.json

Property name | Description | Type | Required? | User friendly name | Example 
--- | --- | --- | --- | --- | --- 
$schema | The URL reference to the schema. | string | yes |  | 
schema_version | The version number of the schema in major.minor.patch format. | string | no |  | 4.6.1
schema_type | The type of the metadata schema entity. | string | yes |  | 
biomaterial_core | Core biomaterial-level information. |  | yes |  | 
cell_morphology | Features relating to the morphology of cells in a biomaterial. |  | no | Cell morphology | 
growth_conditions | Features relating to the growth and/or maintenance of a biomaterial. |  | no | Growth conditions | 
genus_species | The scientific binomial name for the species of the biomaterial. | array | no | Genus species | Homo sapiens
target_cell_type | Cell types expected to be present in the suspension. | array | no | Target cell type | 
total_estimated_cells | Total estimated number of cells in biomaterial. May be 1 for well-based assays. | integer | no | Total estimated cell count | 

## cell_line
_Information about the cell line used in the biomaterial_

Location: type/biomaterial/cell_line.json

Property name | Description | Type | Required? | User friendly name | Example 
--- | --- | --- | --- | --- | --- 
$schema | The URL reference to the schema. | string | yes |  | 
schema_version | The version number of the schema in major.minor.patch format. | string | no |  | 4.6.1
schema_type | The type of the metadata schema entity. | string | yes |  | 
biomaterial_core | Core biomaterial-level information. | string | yes |  | 
catalog_number | The supplier catalogue number for the cell line. | string | no | Catalog number | 
catalog_url | The supplier catalogue URL for the cell line. | string | no | Catalog URL | 
cell_cycle | The cell cycle phase if the cell line is synchronized growing cells or the phase is known. |  | no | Cell cycle | 
cell_line_type | The type of cell line. Must be one of primary, immortalized, stem cell-derived, or synthetic. | string | no | Cell line type | 
cell_morphology | Features relating to the morphology of cells in a biomaterial. |  | no | Cell morphology | 
growth_conditions | Features relating to the growth and/or maintenance of a biomaterial. |  | no | Growth conditions | 
cell_type | The cell type that the cell line was derived from. Should be a CLO ontology. |  | no | Cell type | 
date_established | When the cell line was established, in date-time format. yyyy-mm-ddThh:mm:ssZ. | string | no | Date established | 
disease | Free text describing any disease association to the cell type. Should be an EFO ontology. |  | no | Disease | 
genus_species | The scientific binomial name for the species of the biomaterial. | array | no | Genus species | Homo sapiens
publications | One or more publications in which the cell line creation was cited. | array | no | Publications | 

## organoid
_Information about an organoid biomaterial._

Location: type/biomaterial/organoid.json

Property name | Description | Type | Required? | User friendly name | Example 
--- | --- | --- | --- | --- | --- 
$schema | The URL reference to the schema. | string | yes |  | 
schema_version | The version number of the schema in major.minor.patch format. | string | no |  | 4.6.1
schema_type | The type of the metadata schema entity. | string | yes |  | 
biomaterial_core | Core biomaterial-level information. |  | yes |  | 
cell_morphology | Features relating to the morphology of cells in a biomaterial. |  | no | Cell morphology | 
growth_conditions | Features relating to the growth and/or maintenance of a biomaterial. |  | no | Growth conditions | 
genus_species | The scientific binomial name for the species of the biomaterial. | array | no | Genus species | Homo sapiens
model_for_organ | Organ for which this organoid is a model system. |  | yes | Organ model | 
organoid_age | Age of the organoid. | number | no | Organoid age | 55
organoid_age_unit | The unit in which organoid age is expressed. Must be one of hour, day, week, month, or year. |  | no | Organoid age unit | 
organoid_type | The type of organoid. Must be one of primary, immortalized, stem cell-derived, or synthetic. | string | no | Organoid type | 

## library_preparation_process
_Information about how a sequencing library was created._

Location: type/process/sequencing/library_preparation_process.json

Property name | Description | Type | Required? | User friendly name | Example 
--- | --- | --- | --- | --- | --- 
$schema | The URL reference to the schema. | string | yes |  | 
schema_version | The version number of the schema in major.minor.patch format. | string | no |  | 4.6.1
schema_type | The type of the metadata schema entity. | string | yes |  | 
process_core | Core process-level information. | string | yes |  | 
cell_barcode | Information about cell identifier barcode. | string | no |  | 
input_molecule | Specific type of nucleic acid molecule from which a sequencing library was prepared. | string | yes | Input molecule | 
library_construction | The general approach for sequencing library construction. | string | yes | Library construction approach | Smart-seq2
end_bias | The type of end bias the library has. Must be one of 3' bias, 5' bias, 3' end, 5' end, or none. | string | yes | End bias | three_prime_bias
primer | Primer used for cDNA synthesis from RNA. Must be either poly-dT or random. | string | no | Primer | 
strand | Library strandedness. | string | yes | Strand | 
spike_in | Name of spike-in kit. | string | no | Spike-in kit | ERCC
spike_in_dilution | Dilution of spike-in. | integer | no | Spike-in dilution | 
umi_barcode | Information about unique molecular identifier (UMI) barcode sequences. | string | no |  | 
process_type | The type of process. Ideally an EFO term. |  | no | Process type | 

## sequencing_process
_Information about how a biomaterial was sequenced._

Location: type/process/sequencing/sequencing_process.json

Property name | Description | Type | Required? | User friendly name | Example 
--- | --- | --- | --- | --- | --- 
$schema | The URL reference to the schema. | string | yes |  | 
schema_version | The version number of the schema in major.minor.patch format. | string | no |  | 4.6.1
schema_type | The type of the metadata schema entity. | string | yes |  | 
process_core | Core process-level information. | string | yes |  | 
instrument_manufacturer_model | The manufacturer and model of the sequencer used. | string | yes | Instrument manufacturer and model | Illumina HiSeq 4000
local_machine_name | Local name for the particular machine on which the biomaterial was sequenced. | string | no | Local machine name | 
paired_ends | Was a paired-end sequencing strategy used? Must be either yes or no. | boolean | yes | Paired ends? | 
insdc_experiment | An INSDC (International Nucleotide Sequence Database Collaboration) experiment accession. Accession must start with DRX, ERX, or SRX. | string | no | INSDC experiment | 
insdc_run | An INSDC (International Nucleotide Sequence Database Collaboration) run accession. Accession must start with DRR, ERR, or SRR. | array | no | INSDC run | 
process_type | The type of process. Ideally an EFO term. |  | no | Process type | 

## analysis_process
_Information about an analysis that was performed on data files._

Location: type/process/analysis/analysis_process.json

Property name | Description | Type | Required? | User friendly name | Example 
--- | --- | --- | --- | --- | --- 
$schema | The URL reference to the schema. | string | yes |  | 
schema_version | The version number of the schema in major.minor.patch format. | string | no |  | 4.6.1
schema_type | The type of the metadata schema entity. | string | yes |  | 
process_core | Core process-level information. | string | no |  | 
process_type | The type of process. Ideally an EFO term. |  | no |  | 
inputs | Input parameters used in the pipeline run, these can be files or string values (settings). | array | yes |  | 
reference_bundle | Bundle containing the reference used in running the pipeline. | string | yes |  | 
tasks | Descriptions of tasks in the workflow. | array | yes |  | 
timestamp_stop_utc | Terminal stop time of the full pipeline. | string | yes |  | 
input_bundles | The input bundles used in this analysis run. | array | yes |  | 
outputs | Output generated by the pipeline run. | array | yes |  | 
computational_method | A URI to a versioned workflow and versioned execution environment in a GA4GH-compliant repository. | string | yes |  | 
timestamp_start_utc | Initial start time of the full pipeline. | string | yes |  | 
analysis_run_type | Indicator of whether the analysis actually ran or was just copied forward as an optimization. | string | yes |  | 
metadata_schema | The version of the metadata schemas used for the JSON files. | string | yes |  | 

## collection_process
_Process by which a biomaterial was collected or transformed from another biomaterial or from a population._

Location: type/process/biomaterial_collection/collection_process.json

Property name | Description | Type | Required? | User friendly name | Example 
--- | --- | --- | --- | --- | --- 
$schema | The URL reference to the schema. | string | yes |  | 
schema_version | The version number of the schema in major.minor.patch format. | string | no |  | 4.6.1
schema_type | The type of the metadata schema entity. | string | yes |  | 
process_core | Core process-level information. |  | yes |  | 
process_type | The type of process. Ideally an EFO term. |  | no | Process type | 
collection_method | How the biomaterial was collected. | string | no | Collection method | 
process_reagents | A list of purchased reagents used in this process. | array | no |  | 

## enrichment_process
_Process by which one biomaterial was produced from another biomaterial to favor a feature or characteristic of interest._

Location: type/process/biomaterial_collection/enrichment_process.json

Property name | Description | Type | Required? | User friendly name | Example 
--- | --- | --- | --- | --- | --- 
$schema | The URL reference to the schema. | string | yes |  | 
schema_version | The version number of the schema in major.minor.patch format. | string | no |  | 4.6.1
schema_type | The type of the metadata schema entity. | string | yes |  | 
process_core | Core process-level information. |  | yes |  | 
enrichment_method | The method by which enrichment was achieved. | string | yes | Enrichment method | FACS
markers | A space-delimited list of markers with +/-. | string | no | Markers | CD4+ CD8-
min_size_selected | Minimum cell or organelle size passing selection, in microns. | number | no | Minimum size selected | 
max_size_selected | Maximum cell or organelle size passing selection, in microns. | number | no | Maximum size selected | 
process_type | The type of process. Ideally an EFO term. |  | no | Process type | 

## dissociation_process
_Contains information on the dissociation process used to separate the cells in a specimen._

Location: type/process/biomaterial_collection/dissociation_process.json

Property name | Description | Type | Required? | User friendly name | Example 
--- | --- | --- | --- | --- | --- 
$schema | The URL reference to the schema. | string | yes |  | 
schema_version | The version number of the schema in major.minor.patch format. | string | no |  | 4.6.1
schema_type | The type of the metadata schema entity. | string | yes |  | 
process_core | Core process-level information. |  | yes |  | 
dissociation_method | How cells or organelles were dissociated. | string | yes | Dissociation method | 
nucleic_acid_source | Source cells or organelles from which nucleic acid molecules were collected. | string | no | Nucleic acid source | 
process_type | The type of process. Ideally an EFO term. |  | no | Process type | 
process_reagents | A list of purchased reagents used in this process. | array | no |  | 

## imaging_process
_Process by which images are captured._

Location: type/process/imaging/imaging_process.json

Property name | Description | Type | Required? | User friendly name | Example 
--- | --- | --- | --- | --- | --- 
$schema | The URL reference to the schema. | string | yes |  | 
schema_version | The version number of the schema in major.minor.patch format. | string | no |  | 4.6.1
schema_type | The type of the metadata schema entity. | string | yes |  | 
process_core | Core process-level information. | string | yes |  | 
embedding | Description of embedding conditions. | string | no | Embedding conditions | 
exposure_time | Exposure time - as a floating point number - in seconds. | number | no | Exposure time | 
field_counts | Number of fields in x and y dimensions. | array | yes | Field count | 
field_microns | Microns covered by a field in x, y, and z. Z includes all focal planes in a single file. | array | no | Field microns | 
field_resolution | x, y, and z (number of focal planes) resolution of an individual field. | array | yes | Field resolution | 
fixation | Description of fixation conditions. | string | no | Fixation | 
microscope | Microscope used for imaging. |  | no | Microscope | 
probes | A file containing information on probe sequence, genes they cover, and colors. | string | yes | Probes | 
process_type | The type of process. Ideally an EFO term. |  | no | Process type | 

